changelogs/109_unify_storage_classes_key_in_fusion_info.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
```

Comment:
I don't beleive this is a bugfix. This should be listed as a change because any existing playbooks that use this dict will fail based on this change.
I'm not exactly sure I completely agree with this change as it is purely cosmetic, but also a breaking change.

Reply:
Agree it's no really a bug fix, it is a breaking change and we should be careful about it. On the other hand, variable names should be consistent through the Ansible Collection. Better correct these things while we can, the sooner the better.

Reply:
I agree that we should make such small changes while we can, without distracting anyone.